lib/locator.js

```diff
@@ -17,10 +17,22 @@ function parseArgv(argv, cliPrefix) {
 module.exports = function({options, env, argv, envPrefix = '', cliPrefix = '--'}) {
     const parsedArgv = parseArgv(argv, cliPrefix);
 
+    function getEnvName(prefixes, subKey) {
+        if (!_.isArray(prefixes)) {
```

Comment:
I would suggest invert (`!`) and avoid usage lodash here…

```javascript
if (typeof prefixes === "string")
```

Reply:
Because of performance reasons?

Reply:
Mainly to clarity and code complexity when we read this in a year…